H_w 2/task_8.rb

```diff
@@ -1,7 +1,12 @@
 #8. There is a hash where keys and values are strings. Modify it: all keys should be symbols and all values should be integers. If you couldn't modify some value set it to nil
 
-hash = {'Dima'=>'30', 'Lena'=>'31'}
+
+hash = {'Dima'=>'30', 'Lena'=>'31','Igor'=> 'boy'}
```

Comment:
and what if your hash would have something like 'Name' => '0' pair in 'before' state?
you'll make it nil soon, right?